homeassistant/components/esphome/light.py

```diff
@@ -342,6 +352,11 @@ def color_temp(self) -> int:
         """Return the CT color value in mireds."""
         return round(self._state.color_temperature)
 
+    @property
+    def color_temp_kelvin(self) -> int:
+        """Return the CT color value in Kelvin."""
+        return mired_to_kelvin(self._state.color_temperature)
```

Comment:
```suggestion
        return round(1000000 / self._state.color_temperature)
```

Reply:
Internal function _mired_to_kelvin using the same computation added in commit 879575d1385c06588fbe2d86f39f22a71506e791